src/client.ts

```diff
@@ -50,6 +50,9 @@ export class DelphinusContract {
   }
 
   async getPastEventsFromSteped(fromBlock: number, toBlock: number, step: number) {
+    if(fromBlock < toBlock) {
```

Comment:
????? fromBlock < toBlock is correct scenario?

Reply:
Yes, it's a mistake.